.editorconfig

```diff
@@ -527,20 +524,23 @@ dotnet_diagnostic.SA1614.severity = none
 # SA1615: Element return value should be documented
 dotnet_diagnostic.SA1615.severity = none
 
-# SA1616: Element return value documentation should have text
-dotnet_diagnostic.SA1616.severity = none
-
 # SA1623: Property summary documentation should match accessors
 dotnet_diagnostic.SA1623.severity = none
 
 # SA1642: Constructor summary documentation should begin with standard text
 dotnet_diagnostic.SA1642.severity = none
 
-# SA1649: File name should match first type name
+## SA1649: File name should match first type name
 dotnet_diagnostic.SA1649.severity = none
 #########################################################
 #########################################################
 
+# SA1011: Closing square brackets should be spaced correctly
+dotnet_diagnostic.SA1011.severity = none
+
+# SA1009: Closing parenthesis should be spaced correctly
+dotnet_diagnostic.SA1009.severity = none
+
```

Comment:
Current version of StyleCop Analyzers has bugs related to `?` and `!` operators which are fixed in version 1.2.0 but it's still beta. So commented these rules out to reduce noise.

Reply:
why not leverage the beta? we are beta ourselves?

Reply:
I can propose that change in a separate PR.